RELEASE_NOTES

```diff
@@ -3,7 +3,45 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
-The latest version of easybuild-easyblocks provides 180 software-specific easyblocks and 29 generic easyblocks.
+The latest version of easybuild-easyblocks provides 179 software-specific easyblocks and 30 generic easyblocks.
+
+v3.0.0 (November 16th 2016)
+---------------------------
+
+feature release
+- backwards incompatible changes:
+  - remove deprecated GenomeAnalysisTK/GATK easyblock (#1001)
+  - remove deprecated 'get_netcdf_module_set_cmds' function from netCDF easyblock (#1015)
+  - remove deprecated 'get_blas_lib' function from LAPACK easyblock (#1016)
+  - remove QLogicMPI easyblock (#1023)
+- new easyblock for installing Anaconda (#950)
+- add generic easyblock for Conda installations (#950)
+- various enhancements, including:
+  - enable use of GCCcore as toolchain for Clang, fail if no GCC prefix is found (#1002)
+  - also build Boost MPI library in parallel (#1005, #1038)
+  - enhance g2clib easyblock to allow to install 1.6.0 or higher (#1006)
+  - update QuantumESPRESSO easyblock to support packaging changes in 6.0 (#1007)
+  - add support to Scons generic easyblock to provide argument to specify installation prefix (#1008)
+  - update IntelBase and imkl easyblocks to handle the 2017 versions of compilers/imkl (#1012)
+  - leverage toolchain.linalg functionality in ScaLAPACK easyblock, use copy_file (#1014)
+  - allow netCDF-C++4 to be used with ESMF (#1019)
+  - update Advisor easyblock to support latest versions (#1021)
+  - update CBLAS easyblock to build with foss toolchain (#1024)
+  - update Gurobi easyblock to use copy_file (#1028)
+  - add support for giving /lib preference over /lib64 & co in GCC installation (#1030, #1035)
+  - enable installation of libliberty by default for binutils (#1030)
```

Comment:
libiberty is proper name? I guess people who care about it wouldn't be too much surprised though